core/services/postgres/event_broadcaster.go

```diff
@@ -245,7 +245,7 @@ func (sub *subscription) interestedIn(event Event) bool {
 
 func (sub *subscription) send(event Event) {
 	sub.queue.Add(event)
-	sub.processQueueWorker.WakeUp()
+	sub.processQueueWorker.WakeUpIfStarted()
```

Comment:
Testable?

Reply:
Yes in theory, I don't know exactly how it can happen though. Which is why I'm leanign towards not merging this "fix" without a failing test to go with it so at least we understand how it happens.